api/models/User.js

```diff
@@ -124,25 +106,11 @@ userSchema.methods.generateToken = function () {
       email: this.email,
     },
     process.env.JWT_SECRET,
-    { expiresIn: eval(process.env.SESSION_EXPIRY) },
+    { expiresIn: (eval(process.env.SESSION_EXPIRY) / 1000) },
   );
```

Comment:
I may just be missing something here but how come we're dividing this by 1000?

Reply:
I think was a bug in the original Librechat code that just was not caught because it did not matter.  With an existing LibreChat install compare the expiration time set with the token and the one inside the token.  They do not match unless you divide by 1000

![image](https://github.com/danny-avila/LibreChat/assets/11950213/1709fef3-1209-40c6-9f02-e690191a8118)